### PR TITLE
CONTRIBUTING: Mention PRs need to meet criteria for merging.

### PR DESCRIPTION
**Pull request by flexibeast**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-For information about how to contribute to the Void documentation, please read the following sections of the Void Handbook:
+To contribute to the Void documentation, please read the following sections of the Void Handbook. Pull requests that do not meet the criteria described in these sections will not be merged.
```
Comment (ericonr) [inline CONTRIBUTING.md:1]: Agree with the sentiment, but would like to mention that we are open to helping make PRs fit it. Or do you think that's clear already?

Comment (flexibeast) [reply CONTRIBUTING.md:1]: To manage our own workload, i'd like to encourage users to meet as many of the criteria as possible before submitting a PR. So how about something like:

> will not be merged. Please try to address as many of the criteria as possible before submitting a PR - that will improve the chances of us being able to include your contribution quickly.

Comment (ericonr) [reply CONTRIBUTING.md:1]: Love it! Would prefer passive voice for the last part: "will improve the chances of your contribution being accepted quickly"

Comment (flexibeast) [reply CONTRIBUTING.md]: Yes, that's better - PR updated.

Comment (ericonr) [reply CONTRIBUTING.md]: Thanks!

**Pull request merged**